operators/bosh-operator/DirectorService.js

```diff
@@ -686,6 +687,12 @@ class DirectorService extends BaseDirectorService {
         utils.maskSensitiveInfo(bindCreds);
         logger.info(`+-> Created binding:${JSON.stringify(bindCreds)}`);
       })
+      .tap(() => {
```

Comment:
should be part of then() block.

Reply:
+1- the entire thing should be in the then block